apps/web/src/pages/EmployeeProfile/components/NextRoleSection/NextRoleSection.tsx

```diff
@@ -548,6 +596,35 @@ const NextRoleSection = ({
                     disabled={fetching}
                   />
                 ) : null}
+                <Checkbox
+                  boundingBox
+                  boundingBoxLabel={intl.formatMessage(
+                    employeeProfileMessages.seniorManagementStatus,
+                  )}
+                  id="isCSuiteRole"
+                  label={intl.formatMessage(
+                    employeeProfileMessages.isChiefDeputyCSuiteRole,
+                  )}
+                  name="isCSuiteRole"
+                />
+                {watchIsCSuiteRole === true ? (
```

Comment:
Same thing here about the check being simplified

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/12975/commits/e94ad56dbe1cf5ce8cfdd54c18942764c30482ba